@here/harp-mapview/lib/MapViewFog.ts

```diff
@@ -6,14 +6,14 @@
 
 import { Theme } from "@here/harp-datasource-protocol";
 import { HighPrecisionLineMaterial, SolidLineMaterial } from "@here/harp-materials";
-import { Fog, Material, Mesh } from "three";
+import { Fog, Material, Mesh, OrthographicCamera, PerspectiveCamera, Scene } from "three";
```

Comment:
Do we need to keep the direct imports?

Reply:
@Astrak: It's you code, i'm just following what we have in code ;)

But i agree `import * ...` is more convenient.